docs/chapters/workflow/release/release-process.rst

```diff
@@ -9,10 +9,36 @@ There are two types of releases, source releases, and binary releases. A binary
 release has all the requirements of a source release and more. This split is for
 licensing reasons, as there are more obligations when doing a binary release.
 
+Binary release
+--------------
+Binaries of pelux are released on each process and can be found under our `downloads page`_.
```

Comment:
"our" is unnecessary.

"can be found under downloads page" should be enough

Reply:
PELUX and try to rewrite the entire sentence, please.